media-video/rtmpdump/rtmpdump-2.4_p20131018.ebuild

```diff
@@ -17,11 +17,11 @@ KEYWORDS="amd64 ~arm hppa ~mips ppc ppc64 x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux
 IUSE="gnutls polarssl ssl"
 
 DEPEND="ssl? (
-		gnutls? ( net-libs/gnutls )
-		polarssl? ( !gnutls? ( >=net-libs/polarssl-0.14.0 ) )
-		!gnutls? ( !polarssl? ( dev-libs/openssl ) )
+		gnutls? ( net-libs/gnutls[${MULTILIB_USEDEP}] )
```

Comment:
When adding multilib deps, you need to also add `>=` for version that introduced multilib support or EAPI 5. This is normally done using [multilib-dep-fixor][1] but in this case you can just copy them over from -9999.

Afterwards, please do `repoman full` scan again to ensure that all the deps are stable as well.

[1]:https://bitbucket.org/mgorny/multilib-dep-fixor